Example/Core/Tests/FIRAppAssociationRegistrationUnitTests.m

```diff
@@ -52,8 +50,8 @@ - (void)testPassObject {
   id result = [FIRAppAssociationRegistration registeredObjectWithHost:host
                                                                   key:kKey
                                                         creationBlock:^id _Nullable() {
-    return obj;
-  }];
+                                                          return obj;
```

Comment:
Is there anyway we don't push block contents all the way to the left, i.e., keep the previous style that block indent counts from the start of the statement? IMO the old code reads better for trailing blocks (like Swift).